Jenkinsfile

```diff
@@ -89,72 +89,62 @@ ansiColor('xterm') {
 
           stage('Static Analysis') {
             sh '''#!/bin/bash -ex
-            source ~/.nvm/nvm.sh
-            nvm use v8.9.1
+            source ~/.nvm/nvm.sh &> /dev/null
+            nvm use v8.9.4
             npm run static-analysis
             '''
           }
 
           stage('Unit Tests') {
             sh '''#!/bin/bash -ex
-            source ~/.nvm/nvm.sh
-            nvm use v8.9.1
+            source ~/.nvm/nvm.sh &> /dev/null
```

Comment:
Good!

Reply:
So turns out that doesn't work. The `bash -x` ensures that everything gets logged.